.github/workflows/workflow-installer.yml

```diff
@@ -100,10 +100,10 @@ jobs:
         if: env.INPUT_TYPE != 'auto-detect'
         working-directory: ./target
         run: |
-          git branch install-default-github-actions 2>/dev/null || :
-          git checkout install-default-github-actions
-          git push --set-upstream origin install-default-github-actions 2>/dev/null || :
-          git pull origin install-default-github-actions
+          git branch chore-install-default-github-actions 2>/dev/null || :
```

Comment:
should be "chore/install-gh-actions" (or similar), but chore is separated via "/"